Mathlib/Topology/MetricSpace/Kuratowski.lean

```diff
@@ -112,21 +112,80 @@ end KuratowskiEmbedding
 
 open TopologicalSpace KuratowskiEmbedding
 
-/-- The Kuratowski embedding is an isometric embedding of a separable metric space in `ℓ^∞(ℝ)`. -/
-def kuratowskiEmbedding (α : Type u) [MetricSpace α] [SeparableSpace α] : α → ℓ_infty_ℝ :=
+/-- The Kuratowski embedding is an isometric embedding of a separable metric space in `ℓ^∞(ℕ, ℝ)`.
+-/
+def kuratowskiEmbedding (α : Type u) [MetricSpace α] [SeparableSpace α] : α → ℓ^∞(ℕ) :=
   Classical.choose (KuratowskiEmbedding.exists_isometric_embedding α)
 #align Kuratowski_embedding kuratowskiEmbedding
 
-/-- The Kuratowski embedding is an isometry. -/
+/--
+    The Kuratowski embedding is an isometry.
+    Theorem 2.1 of Metric Embeddings and Lipschitz Extensions by Assaf Naor
```

Comment:
Could you please add this reference to `references.bib`? You can then cite it here as
`[Assaf Naor, *Metric Embeddings and Lipschitz Extensions*][insert-bibkey-here]`.